i18n/pt-BR.json

```diff
@@ -1400,6 +1400,14 @@
     "translation": "RateLimitSettings não foi configurado corretamente usando VaryByHeader e desativando VaryByRemoteAddr"
   },
   {
+    "id": "api.server.start_server.rate_limiting_memory_store",
+    "translation": "Não foi possível inicializar o rate limiting."
+  },
+  {
+    "id": "api.server.start_server.rate_limiting_memory_store",
```

Comment:
that shouldn't happen, let me try and figure out why is it.

thanks @rodrigocorsi2